aspnetcore/fundamentals/middleware.md

```diff
@@ -47,10 +47,8 @@ You can chain multiple request delegates together with [app.Use](https://docs.mi
 [!code-csharp[Main](middleware/sample/Chain/Startup.cs?name=snippet1)]
 
 >[!WARNING]
->Be careful modifying the `HttpResponse` after invoking `next`, because the response may have already been sent to the client. You can use [HttpResponse.HasStarted](https://docs.microsoft.com/en-us/aspnet/core/api/microsoft.aspnetcore.http.features.httpresponsefeature#Microsoft_AspNetCore_Http_Features_HttpResponseFeature_HasStarted) to check whether the headers have been sent.
-
->[!WARNING]
-> Do **not** call `next.Invoke` after calling a `write` method. A middleware component either produces a response or calls `next.Invoke`, but not both.
+>- Calling `next` may send the response to the client. Changes made to the `HttpResponse` after calling `next` will be lost if the response has been sent. Use [HttpResponse.HasStarted](https://docs.microsoft.com/aspnet/core/api/microsoft.aspnetcore.http.features.httpresponsefeature#Microsoft_AspNetCore_Http_Features_HttpResponseFeature_HasStarted) to check whether the headers have been sent.
+>- Do **not** call `next.Invoke` after calling a `write` method. A middleware component either produces a response or calls `next.Invoke`, but not both.
```

Comment:
Why does this bullet speak of "calling a write method" while the one before speaks of "making changes to the HttpResponse".  Is there a substantive difference between the two that calls for the different language?

Given that `write` is styled as code, perhaps it should be `Write`